src/adapter/src/coord.rs

```diff
@@ -5880,24 +6044,25 @@ impl<S: Append + 'static> Coordinator<S> {
             if timelines.contains_key(&id) {
                 continue;
             }
-            let entry = self.catalog.get_entry(&id);
-            match entry.item() {
-                CatalogItem::Source(source) => {
-                    timelines.insert(id, source.timeline.clone());
-                }
-                CatalogItem::Index(index) => {
-                    ids.push(index.on);
-                }
-                CatalogItem::View(view) => {
-                    ids.extend(view.optimized_expr.depends_on());
-                }
-                CatalogItem::RecordedView(rview) => {
-                    ids.extend(rview.optimized_expr.depends_on());
-                }
-                CatalogItem::Table(table) => {
-                    timelines.insert(id, table.timeline());
+            if let Some(entry) = self.catalog.try_get_entry(&id) {
```

Comment:
What caused the need for this?

Reply:
This is probably an artifact, from splitting the strict serializability PR into two separate PRs. The reason was that sometimes this method was called with a transient ID that was not in the catalog. I'll see if it can be removed from this PR.

Reply:
Ah and it's triggered now because initialize_compute_read_policies calls get_timelines now. Ok.